content/blog/2022-06-29-june-community-gems.md

```diff
@@ -0,0 +1,215 @@
+---
+title: June '22 Community Gems
+date: 2022-06-29
+description: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month:
+  working with the DVC cache, DVC data and remotes, using DVC programmatically,
+  and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month:
+  working with the DVC cache, DVC data and remotes, using DVC programmatically,
+  and more.
+picture: 2022-06-29/june-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/may-22-community-gems/1184
+tags:
+  - DVC Remotes
+  - Pipelines
+  - CML
+  - GCP
+  - Community
+---
+
+## [Is there something like `git add-commit -m "some message"` for DVC?](https://discord.com/channels/485586884165107732/563406153334128681/981498675689828362)
```

Comment:
I'm not sure the question is that clear. Later on, they state: "Sometimes if the updated files are "scattered", and you want to take snapshot of all modified files (which were earlier added/ which dvc started tracking), it would be convenient to have a shorthand for that."

If that's the issue, `dvc commit` would already handle this. As you already wrote below:

> If you already have data tracked by DVC, the
> `dvc commit` command adds all the changes to those files or directories without
> having to name each target.

I don't think the proposed alias below is needed for DVC (it doesn't work quite like Git in this sense).

Reply:
I updated the question to be a little more general. Let me know if it makes more sense.